src/commands/revenue.js

```diff
@@ -100,7 +116,7 @@ class Report {
     // find the earliest date a submission was assigned. This date is needed to
     // get the correct number of days of the period, so we can calculate the
     // average daily earnings for the period.
-    if (this.startDate === '2014-01-01') {
+    if (moment(this.startDate).format('DD') === '01') {
```

Comment:
I don't think there is much you can do. It would have to be calculated on setup. It might be an option.